packages/uswds-core/src/js/start.js

```diff
@@ -15,13 +13,16 @@ const svg4everybody = require("./polyfills/svg4everybody");
 
 uswds.components = components;
 
-domready(() => {
+const initComponents = () => {
   const target = document.body;
   Object.keys(components).forEach((key) => {
     const behavior = components[key];
     behavior.on(target);
   });
   svg4everybody();
-});
+};
+
+window.addEventListener("DOMContentLoaded", initComponents, { once: true });
```

Comment:
Passing `once` is a good practice. Helps with Storybook because otherwise it would fire multiple times on load **or** switching between components.